internal/pkg/cli/flag.go

```diff
@@ -182,6 +183,7 @@ const (
 	yesFlagDescription      = "Skips confirmation prompt."
 	execYesFlagDescription  = "Optional. Whether to update the Session Manager Plugin."
 	jsonFlagDescription     = "Optional. Outputs in JSON format."
+	forceFlagDescription    = "Optional. Whether to force a new deployment of the ECS service. Default is false."
```

Comment:
```suggestion
	forceFlagDescription    = "Optional. Force a new ECS service deployment using the existing image. "
```
I can't remember if we need to write the default value or if cobra does it already for us 💭 if it doesn't can we follow the format: `(default false)`